### DISCUSSION TITLE
Add enviroement varibles to deta

### DISCUSSION THREAD

**Discussion by matiasdev30** [deta/cloud-docs#704] [General]
**A try to add enviroement varibles to deta, but give me this error?**
<img width="290" alt="Captura de ecrã 2023-01-03, às 00 45 29" src="https://user-images.githubusercontent.com/50122963/210285380-5d5bce81-c543-482a-a1bf-5c69a46e3706.png">

**Comment by lemonyte** [comment]
Try removing the spaces around the equal sign.

**Reply by matiasdev30** [reply]
Thanks it's work, but now my api dont recognize my redis dabase in DETA, how can I install REDIS, to my DETA server?

**Reply by paluigi** [reply]
I don't think you can install REDIS in Deta. You can try to use the built in DB (BASE), which seems based on Dynamo DB
